ddtrace/tracer/textmap_test.go

```diff
@@ -942,15 +942,38 @@ func TestEnvVars(t *testing.T) {
 						traceparentHeader: "00-00000000000000001111111111111111-2222222222222222-03",
 						tracestateHeader:  "dd=s:0;o:rum;t.dm:-2;t.usr.id:baz64~~,othervendor=t61rcWkgMzE",
 					},
-					out: []uint64{2459565876494606882, 1},
-					tid: traceIDFrom64Bits(1229782938247303441),
-
+					out:    []uint64{2459565876494606882, 1},
+					tid:    traceIDFrom64Bits(1229782938247303441),
 					origin: "rum",
 					propagatingTags: map[string]string{
-						"_dd.p.dm":     "-2",
+						"_dd.p.dm":     "-0",
```

Comment:
Quick question, why '-0'?

Reply:
This is the bug we're fixing here.
See https://github.com/DataDog/system-tests/blob/21687fc490bf326863fee1b738af3671da72bc97/parametric/test_headers_tracestate_dd.py#L509 and the RFC for more details.
> If the traceparent sampled flag equals 1, and the sampling priority is not present or less than or equal to 0, update the sampling priority field to 1, and also set the decision maker tag to DEFAULT.